### PR TITLE
Update middleware.gif

### PR DESCRIPTION
**Pull request by wachunei**
Previous gif was made with generators/yield.

### CODE REVIEW

**Comment by wachunei** [comment]
I just realized markdown guide should be updated to reflect animation frames.

**Comment by iyuq** [comment]
just pull to your branch to fix it

File: docs/guide.md [hunk 1]
```diff
@@ -26,37 +26,18 @@ app.use(responseTime);
   while any code after is the "bubble" phase. This crude gif illustrates how async function allow us
   to properly utilize stack flow to implement request and response flows:
 
-![koa middleware](/docs/middleware.gif)
+![koa middleware](/docs/middleware.gif =100x)
```
Comment (jonathanong) [inline docs/guide.md]: this doesn't work in GitHub markdown preview for me

Comment (jonathanong) [reply docs/guide.md]: what's the `=100x` for?

Comment (wachunei) [reply docs/guide.md]: It's used on other editors, I was trying if it was working on github and forgot to remove it

**Pull request merged**